install.sh

```diff
@@ -446,18 +470,36 @@ arch() {
 }
 
 command_exists() {
-  command -v "$@" > /dev/null 2>&1
+  sh_f command -v "$@" > /dev/null
 }
 
 sh_c() {
-  echoh "+ $*"
+  if [ "$SSH_ARGS" ]; then
+    echoh "+ ssh "$SSH_ARGS" $*"
+  else
+    echoh "+ $*"
+  fi
   if [ ! "${DRY_RUN-}" ]; then
+    sh_f "$@"
+  fi
+}
+
+# Always runs.
+sh_f() {
+  if [ "$SSH_ARGS" ]; then
+    mkdir -p ~/.ssh/sockets
```

Comment:
The ``~/.ssh`` directory requires to have 700 for openssh to accept it. It should probably be chmoded to that.

Reply:
Good catch!